src/firefly/js/visualize/PlotViewUtil.js

```diff
@@ -584,6 +585,44 @@ export function getCenterOfProjection(plot) {
     return plot && makeWorldPt(plot.projection.header.crval1, plot.projection.header.crval2,  plot.imageCoordSys);
 }
 
+
+/**
+ * Find the point in the plot that is at the center of the display.
+ * The point returned is in ImagePt coordinates.
+ * We return it in and ImagePt not screen because if the plot is zoomed the image point will still
+ * be what we want in the center. A alternate scrollX and scrollY may be passed otherwise the current
+ * scrollX and scrollY is used.
+ *
+ * @param {PlotView} plotView
+ * @param {number} [scrollX] optional scrollX, if not defined use plotView.scrollX
+ * @param {number} [scrollY] optional scrollY, if not defined use plotView.scrollY
+ * @return {ImagePt} the center point
+ */
+export function findCurrentCenterPoint(plotView,scrollX,scrollY) {
+    const plot= primePlot(plotView);
+    if (!plot) return null;
+    const {viewDim}= plotView;
+
+    let cc;
+    if (!isUndefined(scrollX) && isUndefined(!scrollY)) {
```

Comment:
not sure what "!scrollY" inside isUndefined means? do you mean !isUndefined(scrollY) ?

Reply:
You right.  This is an old function that I just moved.  After doing some searching I found that the optional parameters were never passed in any of the usages.  That is why it was working.

Reply:
You right. This is an old function that I just moved. After doing some searching I found that the optional parameters were never passed in any of the usages. That is why it was working.